frontend/src/app/main/main.module.ts

```diff
@@ -28,6 +29,7 @@ import { TimeTransitionComponent } from './components/time-transition/time-trans
     ReactiveFormsModule,
     MatButtonModule,
     MatProgressSpinnerModule,
+    AppRoutingModule,
```

Comment:
Declaration array is too long, please split it 
```suggestion
    Declarations: [
         ....
    ]
```